lib/main.coffee

```diff
@@ -2,26 +2,33 @@
 {find} = helpers = require 'atom-linter'
 path = require 'path'
 globule = require 'globule'
+ChildProcess = require('child_process')
+prefixPath = null
+fixPath = require 'fix-path'
```

Comment:
I thought as much, was driving me round in circles! :smile: 

Thanks, i'll update now with some tests too